### ISSUE TITLE
brewkit broken {{version}}

### ISSUE THREAD

**Issue by ddobrinskiy** [teaxyz/cli#426]
@mxcl something looks broken in handling the versions, see last part of semver

![image](https://user-images.githubusercontent.com/29500178/223118444-d57b8fbe-fa54-4676-82ef-b2f1a5d29093.png)

source code to reproduce: https://github.com/teaxyz/pantry.extra/pull/463/commits/6d4cd3fa65fd02a98de90a39764e9c45cd67fcb9

**Comment by mxcl** [comment]
Yeah this is a bug in our curl handling, reopening.

**Comment by mxcl** [comment]
Yeah actually I cannot figure out why it doesn’t error cleanly. Is is possible the URL you downloaded returns a HTTP 200 even though it didn’t actually provide a tarball?

**Comment by ddobrinskiy** [comment]
@mxcl yes, it indeed returned 200

https://github.com/quarto-dev/quarto-cli/releases/tag/v1.3.242
